shared/src/commonTest/kotlin/InstrumentedTestsSuite.kt

```diff
@@ -0,0 +1,319 @@
+                    onNodeWithText(getString(Res.string.button_label_accept)).performClick()
+                    waitUntilDoesNotExist(
+                        hasText(getString(Res.string.button_label_accept)),
+                        10000
+                    )
+
+                    onNodeWithContentDescription(getString(Res.string.content_description_portrait)).assertHeightIsAtLeast(
+                        1.dp
+                    )
+                    onNodeWithText("XXXÉliás XXXTörőcsik").assertExists()
+                    onNodeWithText("11.10.1965").assertExists()
+
+                    onNodeWithText(getString(Res.string.button_label_details)).performClick()
+                    waitUntilExactlyOneExists(
+                        hasText(getString(Res.string.section_heading_age_data)),
+                        3000
+                    )
+                    onNodeWithText("≥14").assertExists()
+                    onNodeWithText("≥16").assertExists()
+                    onNodeWithText("≥18").assertExists()
+                    onNodeWithText("≥21").assertExists()
+                    onNodeWithText("Testgasse 1a-2b/Stg. 3c-4d/D6").assertExists()
+                    onNodeWithText("0088 Testort A").assertExists()
+
+                    onNodeWithText(getString(Res.string.button_label_details)).performClick()
+
+
+                    val client = HttpClient() {
+                        expectSuccess = true
+                        install(ContentNegotiation) {
+                            json()
+                        }
+                    }
+
+
+                    val responseGenerateRequest =
+                        client.post("https://apps.egiz.gv.at/customverifier/transaction/create") {
+                            contentType(ContentType.Application.Json)
+                            setBody(request)
+                        }.body<JsonObject>()
+
+
+                    val qrCodeUrl = responseGenerateRequest["qrCodes"]?.let { qrCodesElement ->
+                        qrCodesElement.jsonArray.firstOrNull()?.jsonObject?.get("url")?.jsonPrimitive?.content
+                    }
+                    val id = responseGenerateRequest["id"]?.jsonPrimitive?.content
+
+                    appLink.value = qrCodeUrl
+
+                    waitUntilExactlyOneExists(
+                        hasText(getString(Res.string.button_label_continue)),
+                        10000
+                    )
+
+                    onNodeWithText(getString(Res.string.button_label_continue)).performClick()
+
+                    val url = "https://apps.egiz.gv.at/customverifier/customer-success.html?id=$id"
+                    val responseSuccess = client.get(url)
+                    assertTrue { responseSuccess.status.value in 200..299 }
+                }
+            }
+        }
+    }
+    })
+
+    val request = Json.encodeToString(RequestBody(
+        listOf(Credential(
+            "urn:eu.europa.ec.eudi:pid:1",
```

Comment:
But in https://github.com/a-sit-plus/valera/pull/145/files#diff-e5f09df452764e1a85d38762cbbb759f8a576eb718b23d754e026ec957f5b5a5R178 a credential with `IdAustriaScheme` is issued into the wallet... why is here a PID requested?

(Please also use constant from `IdAustriaScheme`)